docs/eigenlayer/avs-guides/how-to-build-an-avs.md

```diff
@@ -85,6 +84,8 @@ Please see the example from hello-world-avs [here](https://github.com/Layr-Labs/
 
 6.  Follow the [Testnet Dashboard Onboarding instructions](https://docs.eigenlayer.xyz/eigenlayer/avs-guides/avs-dashboard-onboarding).
 
+7. Implement Rewards payments per the instructions [here](./rewards.md).
```

Comment:
Nit: Only use rewards instead of payments

Reply:
Good call, will draft a correction to r/payments/distributions